data-transformers/transformers/statsTransformer.py

```diff
@@ -159,10 +218,45 @@ def main():
         print("Could not save on alluxio file "+trialID+"_"+containerID+"_environment_data")
     query.saveToCassandra(cassandraKeyspace, statsTable, ttl=timedelta(hours=1))
     
+    
+    # Saving IO Data
+    ####################
     f = lambda a: createIODict(a, trialID, experimentID, containerID, hostID)
     query = data.map(f).reduce(lambda a, b: a+b)
     query = sc.parallelize(query)
     query.saveToCassandra(cassandraKeyspace, ioTable, ttl=timedelta(hours=1))
     
+    
+    #Saving Network data
+    ####################
+    try:
+        networkDataAvailable = len(json.loads(stats[0].decode())["networks"]) != 0
+    except:
+        networkDataAvailable = False
+    
+    if networkDataAvailable:
+        f = lambda a: createNetworkDict(a, trialID, experimentID, containerID, hostID)
+        query = statsRDD.map(f).reduce(lambda a, b: a+b)
+        try: 
```

Comment:
This block is the same as the one in the `else` part of the statement. You can place it after the `if..else` block then.